src/utils/isObject.ts

```diff
@@ -0,0 +1,5 @@
+function isObject(value: unknown): value is Record<string, unknown> {
```

Comment:
Currently, I see that we are using this function from Lodash. Should we use Lodash function like other places?